dockerfiles/init/modules/che/templates/che.env.erb

```diff
@@ -23,7 +22,8 @@ JPDA_ADDRESS=<%= scope.lookupvar('che::che_debug_port') %>
 
 CHE_DATABASE=/data
 CHE_WORKSPACE_STORAGE=/data/workspaces
-CHE_WORKSPACE_LOGS=/data/logs/machine
+CHE_LOGS_DIR=/logs
```

Comment:
No, just thinking about changed behavior [here](https://github.com/eclipse/che/pull/3926/files#diff-1da4013f60e1b92b5613cf9680423a88R63). But now I see that this line sets [that one](https://github.com/eclipse/che/pull/3926/files#diff-1da4013f60e1b92b5613cf9680423a88R63).